lib/base_helper.rb

```diff
@@ -111,6 +113,9 @@ def rand_str(length=8, compat=:nospace_sane)
         return result
       end
 
+      def sha1_password(username: nil, password: nil)
+        return "#{username}:{SHA}" + Base64.encode64(Digest::SHA1.digest(password)).strip()
+      end
```

Comment:
Missed this first review, sorry. I'd only request this moved to `lib/ssl.rb`.

Reply:
This is not openssl related, so I'm not sure if it belongs in this module

Reply:
@akostadinov, I ran it again and don't see any `deprecated` warning from the [cli run](https://openshift-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Runner-v3-smoke/239/consoleFull).  Let me know if you insist on using OpenSSL instead of Digest ruby gem.